src/DynamoCoreWpf/Extensions/MenuBarTypeExtensions.cs

```diff
@@ -23,6 +23,7 @@ internal static string ToDisplayString(this MenuBarType type)
                 case MenuBarType.View: return Properties.Resources.DynamoViewViewMenu;
                 case MenuBarType.Help: return Properties.Resources.DynamoViewHelpMenu;
                 case MenuBarType.Packages: return Properties.Resources.DynamoViewPackageMenu;
```

Comment:
Can you add a TODO to the cases above mentioning these will be deleted in Dynamo 3.0?